http/headers/Permissions-Policy.json

```
@@ -18,10 +18,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "alternative_name": "Feature-Policy",
-              "version_added": "74",
-              "partial_implementation": true,
-              "notes": "Header not recognized but policy can be set through the <code>allow</code> attribute on <code>&lt;iframe&gt;</code> elements (see <a href='https://bugzil.la/1694922'>bug 1694922</a>)"
+              "version_added": false
```

For Firefox, we could add this impl_url:

```suggestion
              "version_added": false,
              "impl_url": "https://bugzilla.mozilla.org/show_bug.cgi?id=Permissions-Policy"
```